docs/rules/valid-expect.md

```diff
@@ -3,8 +3,14 @@
 💼 This rule is enabled in the ✅ `recommended`
 [config](https://github.com/jest-community/eslint-plugin-jest/blob/main/README.md#shareable-configurations).
 
+🔧 This rule is automatically fixable by the
+[`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+
 <!-- end auto-generated rule header -->
 
+Note: Test function will be fixed if it is async and does not have await in the
+async assertion.
```

Comment:
we could use [fancy notes](https://github.com/orgs/community/discussions/16925) here

Reply:
nice! I made the fix in commit ee54e6a.